src/main/User/Requests/UserCreateRequest.java

```diff
@@ -83,6 +88,8 @@
 
   public String getPassword() { return password; }
 
+  public HashMap<String, String> getDefaultIds() { return defaultIds; }
```

Comment:
just wondering, what is the key and value here?

Reply:
It's supposed to just return the dictionary (which can be empty) and the appropriate key will be indexed from this if needed. Would you prefer it if I added a field for the key?